### DISCUSSION TITLE
Nidhogg Discussions

### DISCUSSION THREAD

**Discussion by Idov31** [Idov31/Nidhogg#15] [Announcements]
## 👋 Welcome!
  I'm using discussions as a place to hear what you think about Nidhogg, answer questions and get cool features suggestions.

Here, you can:
  * Ask questions you’re wondering about.
  * Give a feature request.
  * Tell your opinion about the current features and how can they be better.

**Comment by Idov31** [comment]
# Version 0.3 is live! 🥳

After a long time without any new major version, version 0.3 with many improvements and some new features is officially published.
You can see the exact release information [here](https://github.com/Idov31/Nidhogg/releases/tag/v0.3)

**Comment by Idov31** [comment]
# Opening Issues

When you are opening an issue, please use the built in format.
From now on, any issue that won't be in the format will not be addressed.

**Comment by Idov31** [comment]
# Nidhogg wiki is live!

You can see examples and explanations for each and every feature at: https://github.com/Idov31/Nidhogg/wiki

**Comment by ChatGPT-Pro** [comment]
Is this project still maintained?

**Reply by Idov31** [reply]
Yes it is :)
I haven't update in a while (2 months or so) because I'm currently working on other projects, but it is in my plan for the next few months to completely rewrite the client so it'll be easier to use, maybe refactor some of the code and add some new features.
